### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-python-library.git`](https://github.com/cisagov/skeleton-python-library.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/pca-gophish-composition.git pca-gophish-composition
    cd pca-gophish-composition
    git remote add skeleton https://github.com/cisagov/skeleton-python-library.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/CODEOWNERS .pre-commit-config.yaml setup.py src/example/__init__.py 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -59,11 +58,13 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ${{ env.PIP_CACHE_DIR }}
-          key: "test-${{ runner.os }}-py${{ matrix.python-version }}-\
+          key: "test-${{ runner.os }}-\
+            py${{ matrix.python-version }}-\
```
Comment (jsf9k) [inline .github/workflows/build.yml:62]: I don't think these quotes are necessary.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -93,10 +94,12 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ${{ env.PIP_CACHE_DIR }}
-          key: "build-${{ runner.os }}-py${{ matrix.python-version }}-\
+          key: "build-${{ runner.os }}-\
+            py${{ matrix.python-version }}-\
```
Comment (jsf9k) [inline .github/workflows/build.yml:98]: Quotes again

**Review by jsf9k — CHANGES_REQUESTED** [review]
I think there are some unnecessary quotes here.  Not sure if they should be fixed here or upstream.

File: .github/workflows/build.yml (hunk 1 above)
Comment (mcdonnnj) [reply .github/workflows/build.yml:62]: The quotes are needed cleanly turn it into a single string.

Pulled from [cisagov/skeleton-generic](https://github.com/cisagov/skeleton-generic) GHA runs:

```console
key: lint-Linux-py3.9.0-4d6c2ba2786097c74dca7abc3707d576242350073cf738b69537399d20cd487c-c2a90360b9139caea83bba3d7e2739e350b7e3aa53efe7922108b2b44325c3ee-9ff6996b601c7d804757a92c10ff7756d543bded97848e5705c3034c17535606
```

vs

```console
key: lint-Linux-\ py3.9.0-\ 4d6c2ba2786097c74dca7abc3707d576242350073cf738b69537399d20cd487c-\ c2a90360b9139caea83bba3d7e2739e350b7e3aa53efe7922108b2b44325c3ee-\ ac3b46e98deceeb8fd1f101d93a2b45598b68a7975471a0d63d052e0b2f20ca8
```

Comment (mcdonnnj) [reply .github/workflows/build.yml:62]: Adding on, when changed to a block scalar it comes out as:

```console
key: lint-Linux-\
  py3.9.0-\
  4d6c2ba2786097c74dca7abc3707d576242350073cf738b69537399d20cd487c-\
  c2a90360b9139caea83bba3d7e2739e350b7e3aa53efe7922108b2b44325c3ee-\
  ac3b46e98deceeb8fd1f101d93a2b45598b68a7975471a0d63d052e0b2f20ca8"
```

So we may want to codify on using quotation marks honestly.

Comment (jsf9k) [reply .github/workflows/build.yml:62]: OK.  At a minimum I can defer this discussion since it's an upstream decision.

**Review by jsf9k — APPROVED** [review]

File: setup.py [hunk 3]
```diff
@@ -70,7 +81,7 @@ def package_vars(version_file):
     package_dir={"": "src"},
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
     include_package_data=True,
-    install_requires=["docopt", "gophish"],
+    install_requires=["docopt", "gophish", "schema", "setuptools >= 24.2.0"],
```
Comment (mcdonnnj) [inline setup.py]: The `schema` package is not used in this repository, so it should not be added to `install_requires`.

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
🤖 but with a requested change.

File: setup.py (hunk 3 above)
Comment (jsf9k) [reply setup.py]: Eagle eyes!  Removed in commit 8381a17.

**Review by mcdonnnj — APPROVED** [review]
🌟

File: setup.py (hunk 3 above)
Comment (mcdonnnj) [reply setup.py]: Thanks!

**Review by hillaryj — APPROVED** [review]
🌮

**Pull request merged**